Makefile

```diff
@@ -48,7 +48,6 @@ testv:
 
 coverage:
 	${GOBIN} test -covermode atomic -coverprofile cover.out ./src/sybil
-	sed -i "s|_${ROOT_DIR}|.|"	cover.out
```

Comment:
this is for me and my non standard GOPATH, don't remove please

Reply:
I think it was messing up codecov reports, I'll remove for now and investigate more.